tests/bootstrap.php

```diff
@@ -24,6 +24,7 @@
  */
 function _manually_load_plugin() {
 	$_SERVER['HTTP_REFERER'] = 'https://' . $_SERVER['HTTP_HOST']; // phpcs:ignore
+	$_SERVER['HTTP_USER_AGENT'] = 'Mozilla\/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/89.0.4389.90 Safari\/537.36'; // phpcs:ignore
```

Comment:
Do we want to consider a unit test for this? Maybe a control test with this spoofed user agent, and then another test with one of the crawler user agents to ensure that the data isn't written? Not sure if necessary.

Reply:
That would require a more end-to-end test setup, since we're just testing the API handlers in isolation now. But I've added a basic test for `is_a_web_crawler` method in fd2c1ef2b712d99fd744a1546bb8b1a80712939b